Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -490,6 +490,26 @@ public NodejsProjectNode(NodejsProjectPackage package)
             if (CommonUtils.IsSubpathOf(_intermediateOutputPath, fileNode.Url)) {
                 return false;
             }
+

```

Comment:
This entire block should be placed below the checks for _maxFileSize and _analysisIgnoredDirs, not above.